build.sbt

```diff
@@ -105,7 +105,25 @@ lazy val `sparksql-scalapb` = (projectMatrix in file("sparksql-scalapb"))
       scalapbPlugin(scalapb.value.scalapbVersion) -> (Test / sourceManaged).value
     ),
     Test / run / fork := true,
-    Test / javaOptions ++= Seq("-Xmx2G")
+    Test / fork := true,
```

Comment:
Can this be moved and explored on a different PR?

Reply:
removed.

Reply:
@thesamet friendly bump